src/lib/Context.ts

```
@@ -147,19 +149,20 @@ export const createContext = async (configPath: string, overrides: ConfigJson, i
 
   const payer = config.payer ? await createSignerFromPath(config.payer) : signer
 
-  const umi = createUmi(config.rpcUrl! , {
+  const umi = createUmi(config.rpcUrl!, {
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)